src/components/views/BrickGameView/index.tsx

```diff
@@ -25,7 +25,8 @@ const Canvas = styled.canvas`
 const BrickGame = () => {
   useMenuHideWindow(ViewOptions.brickGame.id);
   useEffect(() => {
-    Game.init();
+    const game = new Game();
```

Comment:
Ah this might actually improve the performance quite a bit. Previously the game never unmounted once it started